sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -105,10 +107,140 @@ describe("[AAD] TextAnalyticsClient", function() {
         })
       );
       const allInputs = enInputs.concat(esInputs);
-
       const results = await client.analyzeSentiment(allInputs);
       assert.equal(results.length, testDataEn.length + testDataEs.length);
       assertAllSuccess(results);
+      results.map((result) =>
+        (result as AnalyzeSentimentSuccessResult).sentences.map((sentence) =>
+          assert.isUndefined(sentence.minedOpinions)
+        )
+      );
+    });
+
+    it("client gets positive mined opinions", async () => {
+      const documents = [
+        {
+          text: "It has a sleek premium aluminum design that makes it beautiful to look at.",
+          id: "0",
+          language: "en"
+        }
+      ];
+      const results: AnalyzeSentimentResultArray = await client.analyzeSentiment(documents, {
+        includeOpinionMining: true
+      });
+      assert.equal(results.length, 1);
+      assertAllSuccess(results);
+      const documentSentiment: AnalyzeSentimentSuccessResult = results[0] as AnalyzeSentimentSuccessResult;
+      documentSentiment.sentences.map((sentence) =>
+        sentence.minedOpinions?.map((opinion) => {
+          const aspect = opinion.aspect;
+          assert.equal("design", aspect.text);
+          assert.equal("positive", aspect.sentiment);
+          assert.isAtLeast(aspect.confidenceScores.positive, 0);
+          assert.equal(aspect.confidenceScores.neutral, 0);
+          assert.isAtLeast(aspect.confidenceScores.negative, 0);
+
+          const sleekOpinion = opinion.opinions[0];
+          assert.equal("sleek", sleekOpinion.text);
+          assert.equal("positive", sleekOpinion.sentiment);
+          assert.isAtLeast(sleekOpinion.confidenceScores.positive, 0);
+          assert.isAtLeast(sleekOpinion.confidenceScores.neutral, 0);
```

Comment:
better have this as `assert.equal(aspect.confidenceScores.neutral, 0);` like in line 141

Reply:
same for every test for neutral

Reply:
done.